ivy/functional/backends/jax/activations.py

```diff
@@ -18,17 +18,25 @@ def gelu(
     *,
     approximate: bool = False,
     out: Optional[JaxArray] = None,
+    complex_mode="jax",
 ) -> JaxArray:
     return jax.nn.gelu(x, approximate)
 
 
 def leaky_relu(
-    x: JaxArray, /, *, alpha: float = 0.2, out: Optional[JaxArray] = None
+    x: JaxArray,
+    /,
+    *,
+    alpha: float = 0.2,
+    out: Optional[JaxArray] = None,
+    complex_mode="jax",
```

Comment:
A minor thing, but `complex_mode` should come before `out`, ideally `out` should be the last argument of the function. Same applies to the other functions and backends